core/exercises.py

```diff
@@ -60,6 +70,15 @@ def match_returns_stdout(func, solution):
 
 
 def clean_result(result):
+    """
+    Clean the result string for display
```

Comment:
I'm finding that a lot of these docstrings are not very helpful and a reader doesn't learn much from them that they couldn't guess from the function/argument names or see from a quick glance at the code. This is a strong example of that.